app/views/v1/analysis_consensus/show.json.jbuilder

```diff
@@ -2,12 +2,14 @@ json.id @response.id
 json.submitted_at @response.submitted_at
 json.is_completed @response.submitted_at.present?
 
+number = 0
 json.categories @categories do |category|
   json.id category.id
   json.name category.name
   json.axis_id category.axis_id
 
   json.questions category.rubric_questions(@rubric) do |question|
+    json.number number += 1
```

Comment:
Probably not, since a) it works and b) it's how it's been done elsewhere, in `app/views/v1/report/consensus_report.json.jbuilder`, `app/views/v1/responses/show.json.jbuilder` and `app/views/v1/responses/show_slimmed.json.jbuilder`.

It's my first time using `jbuilder`, but if I had to guess what's happening here, I'd say sending `number` to `json` is caught by `method_missing` in order to return a method which defines a key in the resulting `JSON` with that very name, `number`. In turn, this returned method is sent `number += 1`, which evaluates to an integer (to be the value for this `number` key) and increments `number` for the next cycle.